docs/reference/commandline/run.md

```diff
@@ -78,13 +78,13 @@ Options:
       --memory-swap string          Swap limit equal to memory plus swap: '-1' to enable unlimited swap
       --memory-swappiness int       Tune container memory swappiness (0 to 100) (default -1).
       --name string                 Assign a name to the container
-      --net string                  Connect a container to a network (default "default")
+      --network-alias=[]            Add network-scoped alias for the container
+      --network="bridge"            Connect a container to a network
```

Comment:
I'm not sure how to handle that, but as we do not deprecate `--net`, both flags should appear here (as this is, at some *really small bits*, the same output as `docker run --help`)